tests/unit/aroi/test_leaders_scoring.py

```
@@ -275,47 +275,111 @@ def test_non_eu_country_count_distinct_vs_relay_count(self):
         # ...but only 2 distinct non-EU countries (breadth metric)
         assert metrics['non_eu_country_count'] == 2
 
-    def test_geographic_achievement_uses_non_eu_countries_only(self):
-        """Regression test (PR #217 Bugbot): the non-EU boards' achievement
-        title must be derived from the operator's NON-EU countries only.
-        An EU-heavy operator with a single US relay must get a North
-        America title, not a 'Europe Champion' title, on the non-EU
-        podium."""
+    def _format_boards(self, relay_specs, stub_bandwidth_formatter):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_format_boards`

(ANN202)